service/error.go

```diff
@@ -2,9 +2,17 @@ package service
 
 import "github.com/giantswarm/microerror"
 
-var invalidConfigError = microerror.New("invalid config")
+var (
+	invalidConfigError  = microerror.New("invalid config")
+	invalidVersionError = microerror.New("invalid version")
+)
```

Comment:
Lets please keep the errors close to the matcher like described in https://github.com/giantswarm/fmt/blob/master/go/general_style_guide.md.

Reply:
Just to be clear it should look like:

```
var invalidConfigError = microerror.New("invalid config")

// IsInvalidConfig asserts invalidConfigError.
func IsInvalidConfig(err error) bool {
	return microerror.Cause(err) == invalidConfigError
}

var invalidVersionError = microerror.New("invalid version")

// IsInvalidVersion asserts invalidVersionError.
func IsInvalidVersion(err error) bool {
	return microerror.Cause(err) == invalidVersionError
}
```